plugins/lookup/passwordstore.py

```diff
@@ -346,10 +345,9 @@ def check_pass(self):
                     if ':' in line:
                         name, value = line.split(':', 1)
                         self.passdict[name.strip()] = value.strip()
-            if (self.backend == 'gopass' or
-                    os.path.isfile(os.path.join(self.paramvals['directory'], self.passname + ".gpg"))
-                    or not self.is_real_pass()):
-                # When using real pass, only accept password as found if there is a .gpg file for it (might be a tree node otherwise)
+            if (os.path.isfile(os.path.join(self.paramvals['directory'], self.passname + ".gpg"))
+                or not (self.backend == 'gopass' or self.is_real_pass())):
+                # Only accept password as found if there is a .gpg file for it (might be a tree node otherwise)
```

Comment:
```suggestion
                # In pass and gopass, only accept password as found if there is a .gpg file for it (might be a tree node otherwise)
```

Reply:
@TheLastProject You need the fix mentioned in https://github.com/ansible-collections/community.general/pull/5020#discussion_r933139771

```diff
--- passwordstore.py~	2022-07-29 11:26:02.000000000 +0000
+++ passwordstore.py	2022-07-29 11:27:51.438206000 +0000
@@ -261,11 +261,11 @@
     def is_real_pass(self):
         if self.realpass is None:
             try:
-                self.passoutput = to_text(
+                passoutput = to_text(
                     check_output2([self.pass_cmd, "--version"], env=self.env),
                     errors='surrogate_or_strict'
                 )
-                self.realpass = 'pass: the standard unix password manager' in self.passoutput
+                self.realpass = 'pass: the standard unix password manager' in passoutput
             except (subprocess.CalledProcessError) as e:
                 raise AnsibleError(e)
```